charts/prometheus/templates/alertmanager/deploy.yaml

```diff
@@ -155,9 +159,9 @@ spec:
       dnsConfig:
 {{ toYaml . | indent 8 }}
     {{- end }}
-    {{- if .Values.alertmanager.securityContext }}
+    {{- if .Values.alertmanager.securityContext.enabled }}
       securityContext:
-{{ toYaml .Values.alertmanager.securityContext | indent 8 }}
+{{ omit .Values.alertmanager.securityContext "enabled" | toYaml | indent 8 }}
```

Comment:
Thanks @CarpathianUA, can we use the same approach as with `containerSecurityContext`, that's to say setting it to `{}` would achieve the desired behavior?

Reply:
I believe either both securityContexts should have an enabled flag, or do what you want when set to `{}` intead of each one having its own mechanism

Reply:
@naseemkullah makes sense, thank you. I've just pushed changes for this.